public/app/plugins/panel/status-history/composable_panelcfg.cue

```diff
@@ -28,15 +28,16 @@ composableKinds: PanelCfg: {
 							ui.OptionsWithLegend
 							ui.OptionsWithTooltip
 							ui.OptionsWithTimezones
-							showValue:  ui.VisibilityMode
-							rowHeight:  number
-							colWidth?:  number
-							alignValue: "center" | *"left" | "right"
```

Comment:
Does align value not make sense in status history?

Reply:
I would say it doesn't make a lot of sense. This property was not used anywhere in the panel, only set in .cue config.